.github/workflows/int_test.yml

```diff
@@ -290,44 +296,53 @@ jobs:
       - name: "Run smoke tests"
         run: |
           az iot ops support create-bundle --svc auto
-          az iot ops support create-bundle --svc mq --mq-traces
-          az iot ops check
-          az iot ops check --pre
-          az iot ops check --post
-          az iot ops check --as-object
-          az iot ops check --svc mq --resources broker brokerlistener diagnosticservice
-          az iot ops mq stats
-          az iot ops mq stats --raw
-          az iot ops mq get-password-hash -p test
+          # az iot ops support create-bundle --svc broker --broker-traces
+          # az iot ops check
+          # az iot ops check --pre
+          # az iot ops check --post
+          # az iot ops check --as-object
+          # az iot ops check --svc broker --resources broker brokerlistener diagnosticservice
+          # az iot ops mq stats
+          # az iot ops mq stats --raw
+          # az iot ops mq get-password-hash -p test
```

Comment:
nit/note for future, this command is also removed (and `ops mq stats` will also be renamed under broker)